range_queries/heavy_light_decomposition.cpp

```diff
@@ -0,0 +1,582 @@
+/**
+ * @file heavy_ligt_decomposition.cpp
+ * @brief [Heavy-Light Decomposition](https://en.wikipedia.org/wiki/Heavy_path_decomposition) implementation
+ * @author: [Aniruthan R](https://github.com/aneee004)
+ * @email: aniruth11052000@gmail.com
+ * @CodeChef: [aneee004](https://www.codechef.com/users/aneee004)
+ * @CodeForces: [aneee](https://codeforces.com/profile/aneee)
```

Comment:
I don't think it's needed to link your e-mail here and other profiles.
Linking the GitHub account is good enough.
```suggestion
 * @brief [Heavy Light Decomposition](https://en.wikipedia.org/wiki/Heavy_path_decomposition) implementation
 * @author: [Aniruthan R](https://github.com/aneee004)
```